src/components/App.js

```diff
@@ -0,0 +1,23 @@
+import React, { Fragment }from 'react';
+import Header from './Header';
+import PhotoAlbum from './PhotoAlbum';
+import Footer from './Footer';
+import 'normalize.css';
+import pho from '../assets/pho.jpg';
+import ramen from '../assets/ramen.jpg';
+import spagetthi from '../assets/spagetthi.jpg';
+import udon from '../assets/udon.jpg';
+
+
+function App() {
+  const photos = [pho, ramen, spagetthi, udon];
+  return (
+    <Fragment>
+      <Header />
+      <PhotoAlbum urls={photos} />
```

Comment:
```suggestion
      <PhotoAlbum photos={photos} />
```
This could be "url" or "photos" (design decision based on what you think your app/components need and what makes sense) but whatever you choose, consistency from parent to child is the key.